nixos/modules/services/torrent/transmission.nix

```diff
@@ -110,6 +104,13 @@ in
   };
 
   config = mkIf cfg.enable {
+    systemd.tmpfiles.rules = [
+      "d '${homeDir}' 0775 transmission transmission - -"
```

Comment:
User must be taken from `services.transmission.user`.

Reply:
(this has been fixed :+1:)